insecure/fixtures.go

```diff
@@ -49,6 +50,60 @@ func MessageFixture(t *testing.T, codec network.Codec, protocol Protocol, conten
 		Protocol:  protocol,
 	}
 
+	m := &Message{
+		Egress: egressMsg,
+	}
+
+	// creates corresponding event of that message that
+	// is sent by attack network to orchestrator.
+	e := &Event{
+		CorruptedNodeId:   originId,
+		Channel:           channel,
+		FlowProtocolEvent: content,
+		Protocol:          protocol,
+		TargetNum:         targetNum,
+		TargetIds:         targetIds,
+	}
+
+	return m, e, unittest.IdentityFixture(unittest.WithNodeID(originId), unittest.WithAddress("localhost:0"))
```

Comment:
can you add a constant DEFAULT_ADDRESS = "localhost:0"

Reply:
added https://github.com/onflow/flow-go/pull/2811/commits/995c7d18ad4c8812f33cf69766460d97ca38477d